cmd/version.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const version string = "v0.1.2"
+var version string = "DEV-BUILD"
```

Comment:
I thought 
``` sh
$ pachelbel version
```

was nicer than
``` sh
$ pachelbel -version
```

And you can't reference the `main` package from `cmd` since that would create a cycle.